src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -76,10 +82,18 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 						errors: result.errors
 					}
 				});
+				editEvent.withAdditionalMeasurements({
+					'elapsedTime': new Date().getTime() - startTime
```

Comment:
Measurements should always include the units of measure in the name (e.g. `elapsedTimeMs`)